src/components/Map.js

```diff
@@ -18,10 +21,36 @@ const Map = () => {
       <div className="map">
         <div className="map__inner">
           <GoogleMapReact
-            bootstrapURLKeys={{ key }}
-            defaultCenter={center}
+            bootstrapURLKeys={{
+              key: 'AIzaSyCbfV0IAdkkGv-9mmuAkUJNzCPPfGRO6v0',
+            }}
+            defaultCenter={centerCoords}
             defaultZoom={zoom}
-          />
+            yesIWantToUseGoogleMapApiInternals={true}
+            onGoogleApiLoaded={handleApiLoaded}
+          >
+            {data?.query?.pages?.length > 0 && loaded
```

Comment:
Wow, love this use of [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)!!!